src/components/composites/ImageSlider/ArrowButton.js

```diff
@@ -0,0 +1,57 @@
+import { makeStyles } from "@mui/styles";
```

Comment:
(this is about motocross.jpg above, I just couldn't leave a comment on it directly)
Try installing ImgBotApp (https://github.com/ImgBotApp) on the repository - it will help optimize image files.